src/Bootstrap.php

```diff
@@ -30,6 +42,12 @@ public function bootstrap($app)
 
         $handler = function (\Closure $handler) {
             return function (Message\RequestInterface $request, array $options) use ($handler) {
+                foreach ($this->excludedDomains as $domain) {
```

Comment:
Need to support plain string format
Maybe better use another property for regexes? (like excludedDomainsRegexes)